src/modules/AuthServer.pm

```diff
@@ -4707,8 +4733,8 @@ sub SetupRemoteForReplication
         }
     }
     y2milestone("Updating remote configuration");
-    SCR->Execute(".ldapserver.commitChanges" );
-    $masterldif = SCR->Execute(".ldapserver.dumpConfDb" );
+    SCR->Execute(".ldapserver.commitChanges", GetLoadModulePath );
+    $masterldif = SCR->Execute(".ldapserver.dumpConfDb", GetLoadModulePath);
```

Comment:
So we are enlarging a file that already has 4000 lines, and it is **completely unnecessary**. All of this is an internal implementation detail of the `.ldapserver` agent, isn't it?